### ISSUE TITLE
Ntopng 3 

### ISSUE THREAD

**Issue by gsanchietti** [NethServer/dev#5341]
BandwidthD is the currently supported bandwidth usage monitor for NS 7, we use a patched fork which writes data inside a SQLite database.
After months of heavy use on production, BandwidthD has been found not being reliable enough: often the database writer thread is stuck and there is no way to recover it (even a restart of the service usually doesn't work).

In the meanwhile ntopng has been reached a new milestone. The new version has many improvements and addresses most of CPU and memory issues of the old release. 

We should:

- [x] have a simple ntopng package without pfring dependencies
- [x] adapt nethserver-ntopng package to ntopng 3
- [x] create a page for the Server Manager to display the hosts which are making most network traffic (top talkers)
- [x] push new translations to Transifex

**Comment by gsanchietti** [comment]
A work in progress version can be already tested from nethserver-testing repository:
```
yum --eanblerepo=nethserver-testing install nethserver-ntopng
```

In nethserver-testing:
- ntopng-3.1.170808-3149.x86_64.rpm
- nethserver-ntopng-1.4.1-1.9.g9d80b83.ns7.noarch.rpm
- ntopng-data-3.1.170808-3149.noarch.rpm
- ntopng-pfring-3.1.170808-3149.x86_64.rpm

**Comment by DavidePrincipi** [comment]
In nethserver-testing:
nethserver-ntopng-1.4.1-1.11.ga85584a.ns7.noarch.rpm

**Comment by gsanchietti** [comment]
Added %dist tag: https://github.com/gsanchietti/ntopng/commit/9d0eba532412d73354cbfd89769f7eeed509b19a

**Comment by gsanchietti** [comment]
In nethserver-testing:
nethserver-ntopng-1.4.1-1.13.g82829bc.ns7.noarch.rpm
ntopng-pcap-3.1.170812-3152.el7.centos.x86_64.rpm
ntopng-data-3.1.170812-3152.el7.centos.noarch.rpm

**Comment by SebastianMB-IT** [comment]
**Tested**

**Comment by gsanchietti** [comment]
Released for 7.4:
- ntopng-data-3.1.170812-3152.el7.centos.noarch.rpm
- ntopng-pcap-3.1.170812-3152.el7.centos.x86_64.rpm
- nethserver-ntopng-2.0.0-1.ns7.noarch.rpm
